_posts/2016-08-01-rspamd-1.3.1.md

```diff
@@ -110,4 +110,4 @@ rule "local" {
 
 ### Other bugfixes and improvements
 
-There are couple of other important bugfixes and improvements including a critical fix for extracting values from the top received header. The test framework has been improved with new functional tests and better integration with [CircleCI](https://circleci.com/gh/vstakhov/rspamd).
\ No newline at end of file
+There are couple of other important bugfixes and improvements including a critical fix for extracting values from the top received header. The test framework has been improved with new functional tests and better integration with [DroneCI](https://ci.rspamd.com/rspamd/rspamd).
```

Comment:
I am not sure if we should rewrite the events of the past like this. It really was CircleCI in 2016.

Reply:
Ah yes okay, I just renamed it.
sorry I should have done something else
should I use the web archive maybe?

Reply:
I don't have a strong opinion here: either option, removing a link or redirecting it to the actual CI is fine to me. I think it's easier to leave the change untouched, as it is still better than a broken link.